appveyor.yml

```diff
@@ -1,4 +1,4 @@
-os: Visual Studio 2022
+os: Visual Studio 2019
```

Comment:
Sorry for that commit (just for testing) but I believe this can be reverted now

Reply:
I'll look at this but may not be able to revert for a few days

Reply:
This change has been removed from the PR